src/App.tsx

```diff
@@ -1,14 +1,66 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { actions as actionsTodos } from './features/todos';
 
 export const App: React.FC = () => {
+  const [loading, setLoading] = useState(false);
+  const status = useAppSelector(state => state.filter.status);
+  const query = useAppSelector(state => state.filter.query);
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector((state) => state.currentTodo);
+  const isSelected = currentTodo !== null;
+
+  const loadTodos = async () => {
+    setLoading(true);
+    let allTodos;
+
+    try {
+      allTodos = await getTodos();
+    } catch (error) {
+      throw new Error('Data loading error');
```

Comment:
You should handle error in the catch block, maybe by setting an error state and showing an error message to the user, not just throwing new Error.